packages/databricks-vscode/src/cli/CliWrapper.ts

```diff
@@ -47,6 +47,24 @@ export class CliWrapper {
         return this.extensionContext.asAbsolutePath("./bin/databricks");
     }
 
+    get loggingArguments(): string[] {
+        if (!workspaceConfigs.loggingEnabled) {
+            return [];
+        }
+        const logFileUri = Uri.joinPath(
+            this.extensionContext.logUri,
+            "databricks-cli-logs.json"
+        );
```

Comment:
Let's fetch this from the Logger module or the LoggerManager class. We want to hold this file and the sdk-extension log file in the same place.